examples/jsm/interactive/HTMLMesh.js

```diff
@@ -521,12 +516,13 @@ function html2canvas( element ) {
 	if ( canvas === undefined ) {
 
 		canvas = document.createElement( 'canvas' );
-		canvas.width = offset.width;
-		canvas.height = offset.height;
 		canvases.set( element, canvas );
 
 	}
 
+	canvas.width = offset.width;
+	canvas.height = offset.height;
```

Comment:
About that change, that's the htmlmesh canvas here, that's not really needed to fix the size and position of a rendered canvas element to this htmlmesh canvas.
It's useful if the dom element changed size, but that's incomplete, you should also scale the plane geometry by a ratio of currentWidth/widthUsedWhenYouCreatedThePlane on x and currentHeight/heightUsedWhenYouCreatedThePlane on y to avoid the render to be stretched. In a private fork of that HTMLMesh.js I also made that two lines change with some additional code to store the previous width and height and recreate the htmlmesh completely if that's different, that's quick and dirty, a better way would be to scale the current plane like I said.

Reply:
Oh I actually did a public draft PR for this one in https://github.com/AdaRoseCannon/aframe-htmlmesh/pull/14 but the changes are tied to the aframe component.